src/content/design/layout/breakpoints.md

```diff
@@ -0,0 +1,10 @@
+---
+collection: Design
+title: Breakpoints
+order: 9
+supportingContent: breakpointListing
+---
+
+# Breakpoints
+
+We design for small screens by default, but adapt our UIs for larger viewports. The table below shows what we've already defined as breakpoints for Swarm Design System. It's unlikely that a design will require a breakpoint we haven't specified, but it's ok to write something custom in the web app code to support a design spec.
```

Comment:
I'd add a note here about how these are "mobile first" style breakpoints. "Default" is all screen sizes. Any other breakpoint is to specify a change at that breakpoint _and larger_.